text/00110-compatibility-strategy.md

```diff
@@ -0,0 +1,364 @@
+---
+feature name: compatibility-strategy
```

Comment:
Title should probably be a bit more scoped "CLI Compatibility Assurance Strategy"

Reply:
👍